app/controllers/api/v0/admins_controller.rb

```diff
@@ -8,6 +8,15 @@ def add_portfolio
         render :json => { :errors => e.message }, :status => :unprocessable_entity
       end
 
+      def edit_portfolio
+        portfolio = Portfolio.find(params.require(:portfolio_id))
+        portfolio.update!(portfolio_params)
+
+        render :json => portfolio
```

Comment:
@syncrou do we need a reload of the object following the update in line 13

Reply:
No, line 13 is populating `portfolio` with the updated object attributes. The tests are proving this as well.